spec/flipper/api/v1/actions/actors_spec.rb

```diff
@@ -0,0 +1,108 @@
+require 'helper'
+
+RSpec.describe Flipper::Api::V1::Actions::Actors do
+  let(:app) { build_api(flipper) }
+  let(:actor) { Flipper::Actor.new('User123') }
+
+  describe 'GET /actors/:flipper_id' do
+    before do
+      flipper[:my_feature_1].enable
+      flipper[:my_feature_2].disable
+      flipper[:my_feature_3].enable_actor(actor)
+    end
+
+    context 'when no feature is specified' do
+      before do
+        get "/actors/#{actor.flipper_id}"
+      end
+
+      it 'responds with success' do
+        expect(last_response.status).to eq(200)
+      end
+
+      it 'returns all features' do
+        expected_response = {
+          'flipper_id' => 'User123',
+          'features' => [
+            {
+              'feature' => 'my_feature_1',
+              'enabled' => true,
+            },
+            {
+              'feature' => 'my_feature_2',
+              'enabled' => false,
+            },
+            {
+              'feature' => 'my_feature_3',
+              'enabled' => true,
+            },
+          ],
+        }
+
+        expect(json_response).to eq(expected_response)
+      end
+    end
+
+    context 'when features are specified' do
+      before do
+        get "/actors/#{actor.flipper_id}", keys: "my_feature_2,my_feature_3"
+      end
+
+      it 'responds with success' do
+        expect(last_response.status).to eq(200)
+      end
+
+      it 'returns all specified features' do
+        expected_response = {
+          'flipper_id' => 'User123',
+          'features' => [
+            {
+              'feature' => 'my_feature_2',
+              'enabled' => false,
+            },
+            {
+              'feature' => 'my_feature_3',
+              'enabled' => true,
+            },
+          ],
+        }
+
+        expect(json_response).to eq(expected_response)
+      end
+    end
+
+    context 'when non-existent features are specified' do
```

Comment:
Flipper's default behavior is to not worry about features that don't exist. They are just not enabled. I'm inclined to say this should do the same. Any features passed that don't exist should still be present in the response, but just be enabled => false. That allows us to remove the feature existence check from the action which reduces network calls (like requesting the set of known features) as well. Thoughts?

Reply:
Yep, I agree.